bin/perf.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <s2n.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "error/s2n_errno.h"
+#include "stuffer/s2n_stuffer.h"
+#include "utils/s2n_endian.h"
+#include "utils/s2n_safety.h"
+#include "common.h"
+
+uint64_t read_uint64(uint8_t *data)
+{
+    uint64_t u = 0;
+    u |= ((uint64_t) data[0]) << (8);
```

Comment:
This is just a comment for my own understanding, why is this line not:
```suggestion
    u |= ((uint64_t) data[0]) << (0);
```

Reply:
It actually should be; just a copy pasta mistake.